app/frontend/src/features/messages/requests/HostRequestListItem.tsx

```diff
@@ -48,6 +49,10 @@ export default function HostRequestListItem({
   const { data: otherUser, isLoading: isOtherUserLoading } = useUser(
     isHost ? hostRequest.fromUserId : hostRequest.toUserId
   );
+  const isUnreadClass =
+    hostRequest.lastSeenMessageId !== hostRequest.latestMessage?.messageId
+      ? classes.unread
+      : "";
```

Comment:
```suggestion
  const isUnread = hostRequest.lastSeenMessageId !== hostRequest.latestMessage?.messageId;
```
When used with `classNames` suggestion below